### PR TITLE
Model and Service for Meeting

### PR DESCRIPTION
**Pull request by RodrasSilva**
Model and Service for Meeting

Missing minute endpoint in service

Waiting for review

### CODE REVIEW

File: frontend/lib/services/meetingService.dart [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+      var queryParameters = {
+        'team' : team,
+        'company' : company,
+        'event' : event
+      };
+
+      Response<String> response = 
+          await dio.get("/meetings", queryParameters : queryParameters);
+
+      if(response.statusCode == 200) {
```
Comment (Dpac99) [inline frontend/lib/services/meetingService.dart:18]: Try to change the response handling like in the new version, with try-catch blocks and custom exceptions (my bad, I only put those changes in master branch today). See [member service](https://github.com/sinfo/deck2/blob/master/frontend/lib/services/memberService.dart)

**Review by Dpac99 — CHANGES_REQUESTED** [review]
Minor changes in service, model looks perfet

File: frontend/lib/services/meetingService.dart (hunk 1 above)
Comment (RodrasSilva) [reply frontend/lib/services/meetingService.dart]: Should be solved

**Comment by RodrasSilva** [comment]
Solved
Please Review 
No problem :))

**Review by Dpac99 — APPROVED** [review]

**Pull request merged**